iceberg/iceberg-handler/src/test/queries/positive/iceberg_major_compaction_single_partition_with_evolution2.q

```diff
@@ -28,37 +28,37 @@ set hive.merge.tezfiles=true;
 create table ice_orc (
     a string
  )
-partitioned by (b bigint)
+partitioned by (b double)
```

Comment:
if you need to test all supported types maybe add a separate test for that and use alter command to change the type.

Reply:
I remember on another PR you mentioned that there are already too many q-tests and precommit runs for too long time. These changes of data types do not change what these tests test, but increase coverage on different partition key data types. Are you ok if I correct field names to make them logically suitable to their new data types and leave the data type changes?

Reply:
i think, we should create an integration test similar to `CompactorOnTezTest `

Reply:
Done